pages/index.js

```diff
@@ -1,13 +1,28 @@
 import fetch from "node-fetch";
 import Stats from "../components/Stats";
 import SEO from "../components/common/SEO";
+import { Component } from "react";
+import Router from "next/router";
+import { LAST_VISITED_COUNTRY } from "../constants.js";
 
-export default ({ stats, countries }) => (
-  <>
-    <SEO />
-    <Stats stats={stats} countries={countries} />
-  </>
-);
+export default class index extends Component {
+  componentDidMount() {
+    const lastVisited = localStorage.getItem(LAST_VISITED_COUNTRY);
+    const valueExistsAndValid =
+      lastVisited && this.props.countries.some(e => e[1].iso2 === lastVisited);
+    if (valueExistsAndValid) {
+      Router.push(`/${lastVisited}`);
+    }
```

Comment:
```js
useEffect(() => {
  const lastVisited = window.localStorage.getItem(LAST_VISITED_COUNTRY);
  const valueExistsAndValid = lastVisited && countries.find(e => e[1].iso2 === lastVisited);

    if (valueExistsAndValid) {
      Router.push(`/${lastVisited}`);
    }
}, [])
```

Reply:
Why did you use find instead of some ? I just need to see if the value exists, I don't need to find it

Reply:
oh I see, yes in that case, you're right going with `some`